ruby-challenges.rb

```diff
@@ -31,3 +51,16 @@
 # Expected output: 'LEARN is not a palindrome'
 is_palindrome3 = 'Rotator'
 # Expected output: 'Rotator is a palindrome'
+
+def pali_yes_no(string)
+   good = string.downcase
+  if good == good.reverse
+    "#{string} is a palindrome"
+  else
+    "#{string} is not palindrome"
+  end
+end
+# haha this one was a little tricky but i got it!! WOOP WOOP
```

Comment:
It definitely takes some adjusting to Ruby's built-in methods but it is so much less code than Javascript would have been!  Great job!! 🎉